### PR TITLE
Document funding sources and how to file a reimbursement

### PR DESCRIPTION
**Pull request by choldgraf**
This documents how we can file reimbursements with NumFocus, and also adds a page documenting the funding currently available to the project.

- closes: https://github.com/jupyterhub/team-compass/issues/646

### CODE REVIEW

File: docs/practices/funding.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+## Who can spend funds
+
+Currently we do not have formal processes for approving funding, and we make decisions about where to spend funds similarly to [our other decision making processes](#prs:when-to-merge).
+
+## Funds available to this project
+
+### CZI Community Strategic Lead grant
+
+We have a grant from CZI to fund a Community Strategic Lead position.
+Find all of the grant materials, budget, etc at [this Google Drive folder](https://drive.google.com/drive/folders/1Qk_6IvMwtS9F2yKzF4-wtrng6IJK7Dy6?usp=share_link)l
```
Comment (minrk) [inline docs/practices/funding.md:12]: Looks like an errant `)l`

**Review by minrk — APPROVED** [review]

File: docs/practices/funding.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+## Who can spend funds
+
+Currently we do not have formal processes for approving funding, and we make decisions about where to spend funds similarly to [our other decision making processes](#prs:when-to-merge).
+
+## Funds available to this project
+
+### CZI Community Strategic Lead grant
+
+We have a grant from CZI to fund a Community Strategic Lead position.
+Find all of the grant materials, budget, etc at [this Google Drive folder](https://drive.google.com/drive/folders/1Qk_6IvMwtS9F2yKzF4-wtrng6IJK7Dy6?usp=share_link).
```
[the remark this answers is not in the document]
Comment (choldgraf) [reply docs/practices/funding.md:12]: agreed - what we should have is a shared google drive that only team members have access to, and maybe different access levels based on steering council vs. contributor etc. However I'm not sure how to take action here without removing access for a bunch of people in the process...

[the remark this answers is not in the document]
Comment (manics) [reply docs/practices/funding.md:12]: An immediate action is to make it read only for everyone with the link but read-write for added users only.

[the remark this answers is not in the document]
Comment (choldgraf) [reply docs/practices/funding.md:12]: done - link now provides "commenter" access but not edit access

**Pull request merged**